go/libraries/doltcore/sqle/enginetest/dolt_server_test.go

```diff
@@ -336,3 +347,92 @@ func newConnection(t *testing.T, serverConfig sqlserver.ServerConfig) (*dbr.Conn
 	sess := conn.NewSession(nil)
 	return conn, sess
 }
+
+func TestDoltServerRunningUnixSocket(t *testing.T) {
+	if runtime.GOOS == "windows" {
+		t.Skip("we define socket flag and unix socket cannot be created on Windows, so it will return error")
```

Comment:
You can keep this skip message short and sweet. Something like... 

```suggestion
		t.Skip("unix sockets not support on Windows")
```